Include/internal/pycore_interp.h

```diff
@@ -401,7 +401,10 @@ PyAPI_FUNC(PyStatus) _PyInterpreterState_New(
 #define RARE_EVENT_INTERP_INC(interp, name) \
     do { \
         /* saturating add */ \
-        if (interp->rare_events.name < UINT8_MAX) interp->rare_events.name++; \
+        int val = FT_ATOMIC_LOAD_UINT8_RELAXED(interp->rare_events.name); \
+        if (val < UINT8_MAX) { \
+            FT_ATOMIC_STORE_UINT8_RELAXED(interp->rare_events.name, val+1); \
+        } \
```

Comment:
I know this as a whole isn't atomic. But it's fine because the rare events counter has no need to be *that* accurate anyways.